src/client/Microsoft.Identity.Client/WsTrust/WsTrustWebRequestManager.cs

```diff
@@ -35,7 +35,7 @@ public async Task<MexDocument> GetMexDocumentAsync(string federationMetadataUrl,
             if (httpResponse.StatusCode != System.Net.HttpStatusCode.OK)
             {
                 string message = string.Format(CultureInfo.CurrentCulture,
-                        MsalErrorMessage.HttpRequestUnsuccessful,
+                        MsalErrorMessage.HttpRequestUnsuccessful + "See aka.ms/msal-net-ropc for more information. ",
```

Comment:
https://aka.ms/msal-net-ropc

because if I'm a beginner, I'll try ROPC first (because that's the only thing I understand), and I might not know what aka.ms means?

Reply:
nit: link doesn't exist yet

Reply:
yeah, i know. it will be exist before we ship... :)

Reply:
Lol. I've linked it to docs.ms as the doc is pretty good. https://aka.ms/msal-net/ropc